boat-scaffold/src/main/templates/boat-angular/package.mustache

```diff
@@ -2,6 +2,7 @@
   "name": "{{{npmName}}}",
   "version": "{{{npmVersion}}}",
   "description": "OpenAPI client for {{{npmName}}}",
+  "specVersion": "{{{specVersion}}}",
```

Comment:
Should we make this conditional so that it only appears if it has a value?

Also do we want to put this inside a `backbase` object to avoid potential clashes if package.json adds an official `specVersion` property in the future for some purpose?

Reply:
https://github.com/Backbase/backbase-openapi-tools/pull/326/commits/811324b29c2b1ab938a6a2722c87353fe4964121

Reply:
Some test failures, will take another look ^